deploy/helm/sumologic/values.yaml

```diff
@@ -462,11 +462,14 @@ sumologic:
         fallbackExporters: []
         table: []
         ## -
-        ##  ## exporter is name of the exporter
-        ##  exporter: sumologic/otlp
-        ##  ## statement is an OTTL condition which defines what data should be send to the exporter
-        ##  ## see routing processor documentation for more details:
-        ##  ## https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/aee4b75100530bce7edbf736fbcf76ac4f6ced6d/processor/routingprocessor/README.md#tech-preview-opentelemetry-transformation-language-statements-as-routing-conditions
+        ##  ## exporters is an array of the exporter
+        ##  exporters:
```

Comment:
Could we add an example in the comment to guide the customer? Also is there a breaking change to how the customer configures additional exporters?